lib/devicetrust/enroll/enroll_test.go

```diff
@@ -16,6 +16,7 @@ package enroll_test
 
 import (
 	"context"
+	"github.com/gravitational/trace"
```

Comment:
nit: Move to the external deps import block.

Reply:
oops, might just need to run `make fix-imports`